about.md

```diff
@@ -0,0 +1,217 @@
+        fullsize: /style/images/docs/about/fullsize/pico_folder.png
+      -
+        heading: Your Content Folder
+        description: An example of a content folder in Pico.  This one is from a writing website.
+        thumbnail: /style/images/docs/about/thumbnails/content_folder.png
+        fullsize: /style/images/docs/about/fullsize/content_folder.png
+      -
+        heading: Pico's Configuration File
+        description: Configuring Pico is as simple as uncommenting a few lines in the included template.
+        thumbnail: /style/images/docs/about/thumbnails/config.png
+        fullsize: /style/images/docs/about/fullsize/config.png
+      -
+        heading: Editing a Page
+        description: Creating content couldn't be easier using Markdown.  Here's an example of editing a page.
+        thumbnail: /style/images/docs/about/thumbnails/editing_markdown.png
+        fullsize: /style/images/docs/about/fullsize/editing_markdown.png
+  themes:
+    headline: Community Themes
+    description: Ready-to-go, community developed themes to customize your Pico website.
+    style: carousel-box
+    images:
+      -
+        heading: NotePaper Theme
+        description: "[NotePaper](http://development.sjmcdougall.com/pico-themes/NotePaper/) - A highly customizable theme, designed exclusively for Pico."
+        thumbnail: /style/images/docs/about/thumbnails/notepaper_theme.jpg
+        fullsize: /style/images/docs/about/fullsize//notepaper_theme.jpg
+      -
+        heading: Simple Sidebar Theme
+        description: "[Simple Sidebar](http://startbootstrap.com/template-overviews/simple-sidebar/) is a theme from [Start Bootstrap](http://startbootstrap.com/), [ported to Pico](https://github.com/dmelo/bt-theme)."
+        thumbnail: /style/images/docs/about/thumbnails/simple_sidebar_theme.png
+        fullsize: /style/images/docs/about/fullsize/simple_sidebar_theme.png
+      -
+        heading: SimpleTwo Theme
+        description: "[SimpleTwo](https://github.com/sonst-was/simpleTwo) is a simple theme with two columns for picoCMS."
+        thumbnail: /style/images/docs/about/thumbnails/simpletwo_theme.png
+        fullsize: /style/images/docs/about/fullsize/simpletwo_theme.png
+      -
+        heading: Magazine Theme
+        description: "[Magazine Theme](http://freehtml5.co/preview/?item=magazine-free-html5-bootstrap-template) from [FreeHTML5.co](http://freehtml5.co/), [ported to Pico](https://github.com/BesrourMS/magazine)."
+        thumbnail: /style/images/docs/about/thumbnails/magazine_theme.jpg
+        fullsize: /style/images/docs/about/fullsize/magazine_theme.jpg
+---
+
+Pico is a stupidly simple, blazing fast, flat file CMS.  For those of us who aren't an expert webmaster, what does that even mean?  In the most basic sense, it means that there is no administration backend or database to deal with. You simply create markdown files in the content folder and those files become your pages.  There's *much* more to Pico than that though.
```

Comment:
> Pico is a stupidly simple, blazing fast, flat file CMS.

That has been said in the headline already. Kind of, at least.

> For those of us who aren't an expert webmaster

Even as a "expert webmaster" one could be unaware of what "flat file CMS" means. Just "But what does that even mean?" is better. However, the headline states the same question already :smile: